spec/integration/dry/configurable/setting_spec.rb

```diff
@@ -185,6 +185,68 @@
 
     include_context "configurable behavior"
 
+    context "can be configured with another class's settings" do
```

Comment:
FWIW, this file is actually testing the `.setting` method (n.b. the `RSpec.describe ... ".setting"` call at the top of the file), and these new examples are probably outside of that scope. It'd probably be helpful to move them into a new `spec/.integration/.../settings_spec.rb` file.